addons/pos_restaurant/static/src/app/split_bill_screen/split_bill_screen.xml

```diff
@@ -24,14 +24,23 @@
                     </div>
                     <div class="controls flex-column flex-nowrap w-100 w-lg-50">
                         <div class="order-info mb-2 mt-2 mt-lg-0 py-3 py-lg-4 rounded-3 text-center bg-view">
+                            <div class="fs-4 text-start ms-2">
+                                Total amount: <t t-esc="env.utils.formatCurrency(currentOrder.get_total_with_tax())" />
+                            </div>
                             <span class="subtotal text-success">
                                 <t t-esc="env.utils.formatCurrency(newOrderPrice)" />
                             </span>
+                            <div class="fs-2">
+                                <t t-esc="numberOfProducts"/> product(s) selected
```

Comment:
now that `numberOfProducts` is no longer a getter you have to call it. Maybe you can also rename it to `getNumberOfProducts`. That's way it's clear that you have to call it